bindgen/transitive_repositories.bzl

```diff
@@ -8,7 +8,6 @@ def rust_bindgen_transitive_dependencies():
 
     llvm_configure(
         name = "llvm-project",
-        repo_mapping = {"@llvm_zlib": "@zlib"},
```

Comment:
Why delete this?

Reply:
I've made the `repo_mapping` parameter optional and disabled it when using the Bzlmod extension. Even if Bazel is updated to silently ignore the parameter under Bzlmod, `rules_rust` would have to increase its minimum supported Bazel version to avoid errors.